OlinkAnalyze/R/globals.R

```diff
@@ -1,13 +1,12 @@
-#' Global Variables Defined
-#' @importFrom utils globalVariables
```

Comment:
Since we use globalVariables only with the namespace added (`utils::globalVariables`) this import is not required.